bindings/go/src/examples/perftest/perftest.go

```diff
@@ -272,6 +273,7 @@ func serverStart() error {
 		}
 
 		perfTest.wg.Wait()
+		i += perfTest.numCompletedRequests
```

Comment:
but now we increase i by "numThreads" on each iteration, seems wrong

Reply:
`perfTest.numCompletedRequests` isn't guaranteed to be `numThreads` each iteration since we do `ucp_worker_progress()` on each thread until it returns `0`.

And even if `perfTest.numCompletedRequests` equal to `numThreads`, `commonCompRequests` is the number of all completions for all threads.

Reply:
1. can we launch all threads once, and wait until perfTest.numCompletedRequests reaches commonCompRequests?
2. remove the variable I
3. rename commonCompRequests to totalNumRequests